balancedTree.py

```diff
@@ -0,0 +1,24 @@
+# Approach: In order to determine if a binary tree is balanced or not check the following
+# 1. left subtree of node is balanced
+# 2. right subtree of node is balanced
+# 3. abs difference between left and right subtree is <= 1
+# Time O(N)
```

Comment:
Time: O(n^2), the tree is traversed multiple times in order to calculate height. Please also try the bottom up solution which has a better Time Complexity.